### ISSUE TITLE
Doesn't recognize scss maps anymore

### ISSUE THREAD

**Issue by iGadget73** [abe33/atom-pigments#309]
I'm using scss maps and within those maps it doesn't show the colors anymore.

```scss
$color: (
	t: 		transparent,
	bg: 		$gray,
	background:	$gray,
	s: 		$black,
	w: 		$white,
	r: 		$red,
	header: 	#0a282d,
	main: 		#4bb4aa,
	main-light:	#ebf5f4,
	main-dark: 	#328782
);
```

**Comment by C-Bouthoorn** [comment]
Thanks for your example, but could you please also share your OS and Atom version with us?

**Comment by iGadget73** [comment]
Sure, Mac OS 10.12.5 and atom 1.17, both latest versions.

**Comment by pingshunhuangalex** [comment]
Then hex color highlighting is fixed in the latest `Atom 1.18.0-beta1`, but it seems `pigments` is still not able to recognise scss maps yet.
